src/control/cmd/dmg/system.go

```diff
@@ -69,8 +70,12 @@ func (cmd *leaderQueryCmd) Execute(_ []string) (errOut error) {
 		return cmd.outputJSON(resp, err)
 	}
 
-	cmd.Infof("Current Leader: %s\n   Replica Set: %s\n", resp.Leader,
-		strings.Join(resp.Replicas, ", "))
+	if cmd.ReplicasOff {
+		cmd.Infof("Replicas Off: %s\n", strings.Join(resp.ReplicasOff, ", "))
+	} else {
+		cmd.Infof("Current Leader: %s\n   Replica Set: %s\n  Replicas Off: %s\n", resp.Leader,
+			strings.Join(resp.Replicas, ", "), strings.Join(resp.ReplicasOff, ", "))
```

Comment:
minor: I think "Unresponsive Replicas" might be clearer here.

Reply:
Will update